pom.xml

```diff
@@ -250,7 +250,7 @@
         <dependency>
             <groupId>com.conveyal</groupId>
             <artifactId>gtfs-lib</artifactId>
-            <version>6.0.1</version>
+            <version>6.0.3</version>
```

Comment:
This is why I don't like MTC branches diverging from dev a bunch. We end up reviewing code we've already reviewed.